website/docs/cloud-docs/api-docs/variable-sets.mdx

```diff
@@ -256,120 +292,203 @@ curl \
+      "var-count": 5,
+      "workspace-count": 2,
+      "project-count": 2
+    },
+    "relationships": {
+      "organization": {
+        "data": {
+          "id": "hashicorp",
+          "type": "organizations"
+        }
       },
-      "relationships": {
-          "organization": {
-              "data": {
-                  "id": "hashicorp",
-                  "type": "organizations"
-              }
+      "vars": {
+        "data": [
+          {
+            "id": "var-mMqadSCxZtrQJAv8",
+            "type": "vars"
+          },
+          {
+            "id": "var-hFxUUKSk35QMsRVH",
+            "type": "vars"
+          },
+          {
+            "id": "var-fkd6N48tXRmoaPxH",
+            "type": "vars"
+          },
+          {
+            "id": "var-abcbBMBMWcZw3WiV",
+            "type": "vars"
           },
-          "vars": {
-              "data": [
-                  {
-                      "id": "var-kFW8M9gJ1pBYobSx",
-                      "type": "vars"
-                  },
-                  {
-                      "id": "var-XmEvinZu9nMxUoNt",
-                      "type": "vars"
-                  }
-              ]
+          {
+            "id": "var-vqvRKK1ZoqQCiMwN",
+            "type": "vars"
           }
+        ]
+      },
+      "workspaces": {
+        "data": [
+          {
+            "id": "ws-UohFdKAHUGsQ8Dtf",
+            "type": "workspaces"
+          },
+          {
+            "id": "ws-XhGhaaCrsx9ATson",
+            "type": "workspaces"
+          }
+        ]
+      },
+      "projects": {
+        "data": [
+          {
+            "id": "prj-1JMwvPHFsdpsPhnt",
+            "type": "projects"
+          },
+          {
+            "id": "prj-SLDGqbYqELXE1obp",
+            "type": "projects"
+          }
+        ]
       }
+    }
   }
 }
 ```
 
-## List Variable Set
+## List Variable Sets
 
 List all variable sets for an organization.
 
 `GET organizations/:organization_name/varsets`
 
-| Parameter            | Description                                            |
-| -------------------- | ------------------------------------------------------ |
-| `:organization_name` | The name of the organization the workspace belongs to. |
+| Parameter            | Description                                              |
+|----------------------|----------------------------------------------------------|
+| `:organization_name` | The name of the organization the variable sets belong to |
+
+List all variable sets for a project. This includes global variable sets from the project's organization.
 
-List all variable sets for a workspace.
+`GET projects/:project_id/varsets`
+
+| Parameter     | Description    |
+|---------------|----------------|
+| `:project_id` | The project ID |
+
+List all variable sets for a workspace. This includes global variable sets from the workspace's organization and variable sets
+attached to the workspace's parent project.
```

Comment:
From what I've seen, we usually say workspaces are "contained within" projects:
```suggestion
attached to the project this workspace is contained within.
```
Please feel free to push back if you don't like this phrasing!